requirements.txt

```diff
@@ -8,7 +8,7 @@ eventlet==0.23.0
 flex==6.13.2
 git+https://github.com/Kami/logshipper.git@stackstorm_patched#egg=logshipper
 git+https://github.com/StackStorm/mongoengine.git@stackstorm_patched#egg=mongoengine
-git+https://github.com/StackStorm/orquesta.git@master#egg=orquesta
+git+https://github.com/StackStorm/orquesta.git@9cba10037dcf66dd44778d9ccb769039edc81237#egg=orquesta
```

Comment:
Is this intentional? Can we use a release tag?

Reply:
@bigmstone it's intentional per @Kami's request. if we don't add/change the revision, orquesta won't get updated in circleci.  we'll do a release for orquesta and use the release tag when we cut the release branch for st2.